assets/src/application/ui/layout/espressoTable/style/index.scss

```diff
@@ -3,7 +3,9 @@
  * © 2020 Event Espresso
  * ------------------------------------------------------------------------- */
 
-@import 'application/ui/styles/mixins/breakpoints.scss';
+@import "application/ui/styles/mixins/breakpoints.scss";
+@import "./row.scss";
+@import "./data-cell.scss";
```

Comment:
good idea  👍 

i bet there's still more styles lurking in that mammoth file that could go here, but not a big deal